subscription.go

```diff
@@ -64,6 +64,7 @@ type SubscriptionResponse struct {
 
 type SubscriptionQueryParam struct {
 	GroupID string
+	AppId   string
```

Comment:
@arajendiran thanks for opening this PR. Can we use `AppID` here for consistency sake?

Reply:
fixed.